src/nvim/diff.c

```diff
@@ -68,10 +96,10 @@ void diff_buf_delete(buf_T *buf)
 
     if (i != DB_COUNT) {
       tp->tp_diffbuf[i] = NULL;
-      tp->tp_diff_invalid = TRUE;
+      tp->tp_diff_invalid = kTrue;
```

Comment:
`true` (bool) not TriState. Same for the other cases.

Reply:
Thanks!